### DISCUSSION TITLE
On KDE with Wayland in Archlinux, virtual display and real desktop should automate to be streamed together.

### DISCUSSION THREAD

**Discussion by Onetopabove** [rustdesk/rustdesk#7352] [Q&A]
Just as this screenshot shows, almost every time I remotely access my Archlinux, I have to manually choose which screen will be shared in the controlled side:
![image](https://github.com/rustdesk/rustdesk/assets/38457754/98ba7992-77e4-4c60-9c75-e0c2c6b72cd8)
The top left one is for mouse control(if without, I can't use mouse), perhaps including keyboard.
The bottom one is the real sreen.
My question is ***if there's any method that every time the PC being controlled, it doesn't bother me to mannually choose the essential displays?*** Especially, sometimes it's inconvenient to do so.
Thanks!!!

**Comment by xlionjuan** [comment]
Wayland still not suitable for unattended control,but is okay for remote assistance, because a person will in front of the computer physically, they're(Wayland) still fixing this, maybe the time that Fedora drop support for X11.

**Comment by mxvin** [comment]
unrelated inquiry for rustdesk. That popup is a KDE app 'portal' for handling Wayland screen capture requests. If you don't want that, avoid using KDE. However, all desktop environments will implement similar security measures for screen capture, as it's the standard approach in Wayland.

**Reply by xlionjuan** [reply]
It is Wayland's problem, NOT KDE.
